pkg/channel/distributed/common/kafka/sarama/sarama_test.go

```diff
@@ -247,30 +237,30 @@ func TestMergeSaramaSettings(t *testing.T) {
 	// Verify error when no Data section is provided
 	configEmpty := commontesting.GetTestSaramaConfigMap(commontesting.NewSaramaConfig, commontesting.TestEKConfig)
 	configEmpty.Data = nil
-	err = MergeSaramaSettings(config, configEmpty)
+	config, err = MergeSaramaSettings(config, configEmpty)
 	assert.NotNil(t, err)
-}
 
-// Verify that a sarama.Config struct can be serialized into a string without causing JSON errors
-func TestMarshalSaramaJSON(t *testing.T) {
-	config := sarama.NewConfig()
-	jsonResult, err := MarshalSaramaJSON(config)
-	assert.Nil(t, err)
-	assert.NotEqual(t, "", jsonResult)
+	// Verify error when an invalid Version is provided
+	configMap := commontesting.GetTestSaramaConfigMap(commontesting.NewSaramaConfig, commontesting.TestEKConfig)
+	regexVersion := regexp.MustCompile(`Version:\s*\d*\.[\d\.]*`)	// Must have at least one period or it will match the "Version: 1" in Net.SASL
```

Comment:
<!--gotool-bot-->
Format Go code:
```suggestion
	regexVersion := regexp.MustCompile(`Version:\s*\d*\.[\d\.]*`) // Must have at least one period or it will match the "Version: 1" in Net.SASL
```